src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -57,6 +57,8 @@ public class DriveTrain extends SubsystemBase {
 
     private static final double DRIVE_BASE_RADIUS_METERS = Math.sqrt(TRACKWIDTH_METERS*TRACKWIDTH_METERS + WHEELBASE_METERS*WHEELBASE_METERS) / 2.0;
 
+    public  static final double  DRIVETRAIN_ANGLE_TOLERANCE = 5;
```

Comment:
Really should not use "DRIVETRAIN_" inside Drivetrain. It is redundant. Also, it would be good use include "_DEGREES" in the name, so we know what units.

Reply:
done